src/server/AppManager.ts

```diff
@@ -170,7 +170,9 @@ export class AppManager {
                 aff.setCompilerErrors(result.compilerErrors);
 
                 if (result.compilerErrors.length > 0) {
-                    throw new Error(`Failed to compile due to ${ result.compilerErrors.length } errors.`);
+                    const errors = result.compilerErrors.map((error) => JSON.stringify(error, null, 4)).join('\n');
```

Comment:
Stringifying the whole error object seems to be very verbose.

Maybe just map all the `error.message`s?

Reply:
resolved in https://github.com/RocketChat/Rocket.Chat.Apps-engine/pull/166/commits/46a1e0db3e09d7d7852862823c1740098778ff0c